### DISCUSSION TITLE
"Song Select" Skin editor don't work.

### DISCUSSION THREAD

**Discussion by xDShnick** [ppy/osu#26833] [Ideas]
### Type

Cosmetic

### Bug description

Can't move existing objects in "Song Select" skin editor. Can create new object and change them, but not existing ones(leaderboard, maps, map details, searchbar, etc.)
p.s. don't know how to make priority:3

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/153918945/1351c01b-e9b7-4b91-bc2d-879e80fc7219)
![image](https://github.com/ppy/osu/assets/153918945/be27e55f-a3c2-4606-9c15-64032be439b7)

### Version

2024.130.2

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14103174/compressed-logs.zip)

**Comment by Stoppedpuma** [comment]
The elements you listed are currently not skinnable. If you're looking for a redesign you can stare at this for the time being https://github.com/ppy/osu/issues/22204.
